src/components/Jobs/Service.ts

```diff
@@ -35,64 +34,41 @@ export const getJobCIPipelines = (jobId: number) => {
     return get(`${Routes.JOB_CI_PIPELINE_LIST}/${jobId}`)
 }
 
-export const getJobsInitData = (payloadParsedFromUrl: Record<string, any>): Promise<any> => {
+export const getJobsInitFilters = (): Promise<JobsMasterFilters> => {
     return Promise.all([getProjectList(), getEnvironmentListMinPublic()]).then(([projectsRes, environmentsRes]) => {
```

Comment:
Please use getEnvironmentListMinPublic from common, and please connect with @eshankvaish, if newly created resource filter hook can be used?

Reply:
will discuss and pick later for resource hook